### PR TITLE
chore: add a CMakeLists.txt

### LINKED ISSUE #18: Add CMakeLists.txt

### PR DESCRIPTION
**Pull request by rpappalax**
r? @andrewgtp 
Closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,27 @@
 The purpose of this repo is to demonstrate algorithms in C based on a fork from Kyle Louden's book Mastering Algorithms in C. 
 
 
-### Setup 
+## Run Examples
```
Comment (andrewgtp) [inline README.md:19]: Perhaps we should replace makefile doc with cmake doc? @rpappalax

**Review by andrewgtp — COMMENTED** [review]
See my comment on readme

File: README.md (hunk 1 above)
Comment (rpappalax) [reply README.md:19]: agreed!  Let's also make sure our CMakeLists.txt works with cmake from the command line.

**Comment by rpappalax** [comment]
README updated!  OK to merge?

**Comment by andrewgtp** [comment]
Looks good! 👍

**Comment by andrewgtp** [comment]
👍

**Pull request merged**